lib/recase/cases/sentence_case.ex

```diff
@@ -1,23 +1,25 @@
 defmodule Recase.SentenceCase do
-    @moduledoc """
-    Module to convert strings to `Sentence case`.
-  
-    This module should not be used directly.
-  
-    Read about `Sentence case` here:
-    https://en.wikipedia.org/wiki/Letter_case#Sentence_case
-    """
+  @moduledoc """
+  Module to convert strings to `Sentence case`.
 
-    alias Recase.SnakeCase
+  This module should not be used directly.
```

Comment:
I am a little bit confused, why I had this warning: `This module should not be used directly.`

I guess it should be removed, since `SentenceCase` is a public module.

Reply:
Eh. I did not invent this wording, it’s how it was stated in `Recase` before.

Reply:
Oh, sorry. My bad.

Reply:
Oh, sorry. My bad.